src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -255,14 +261,11 @@ const LandingPage = () => {
               What if I can’t find all my medical records?
             </h3>
             <p className="vads-u-margin-bottom--2">
-              Right now, only some types of medical records are available here
-              on VA.gov. And your records on VA.gov only include health
-              information your VA providers have entered.
+              Your medical records on VA.gov only include health information
+              your Va providers have entered.
             </p>
             <p className="vads-u-margin-bottom--2">
-              To find other types of medical records
-              <code>&#8212;</code>
-              including health information you entered yourself
+              To find health information you entered yourself,
```

Comment:
These should be subject to the same condition as the "Other Medical Records" section above.

Reply:
fixed.